packages/shared/blockchain/evm-based/src/lib/transactions/custodial/index.ts

```diff
@@ -84,6 +84,9 @@ const custodialWalletBatch = async (
   testnet?: boolean,
   provider?: string,
 ) => {
+  if ('feesCovered' in body) {
+    throw new Error('Use the sdk.custodial.generateCustodialBatch method with feesCovered.')
```

Comment:
much more developer friendly is onvoke this function right now instead of throwing error. @npwork what do you think?

Reply:
What about omit `feesCovered` in `ChainGenerateCustodialWalletBatch`? So it wont be even possible to use this method with that.

Reply:
I have changed it so that the api call gets invoked directly from the sdk call if the parameter exists in body, updated the examples etc., but not 100% sure how to write tests for this - I understand direct api calls should not be performed in tests - are there proper instructions anywhere or somewhere I could look for guidance? @vlnevyhosteny